utils/src/main/kotlin/Utils.kt

```diff
@@ -112,6 +117,31 @@ object OkHttpClientHelper {
 
         return client.newCall(request).execute()
     }
+
+    /**
+     * Call [execute] repeatedly if an [IOException] occurs.
+     *
+     * @param cachePath The path to the local cache.
+     * @param request The HTTP request object.
+     * @param repetitions How often the request is repeated if an [IOException] occurs.
+     * @param timeout The timeout between two repetitions in milliseconds.
+     */
+    fun executeWithRetry(cachePath: String, request: Request, repetitions: Int = 2, timeout: Long = 0): Response {
```

Comment:
OkHttp is already doing this internally it seems, see https://medium.com/inloop/okhttp-is-quietly-retrying-requests-is-your-api-ready-19489ef35ace. The `Builder()` sets `retryOnConnectionFailure = true` by default. So do you have any clue why we would still need this?

Reply:
No, I have checked the sources and think we should drop this.